src/painting/painter.rs

```diff
@@ -95,7 +95,16 @@ impl Painter {
     /// [`Painter::handle_resize()`] instead
     pub(crate) fn initialize_prompt_position(&mut self) -> Result<()> {
         // Update the terminal size
-        self.terminal_size = terminal::size()?;
+        self.terminal_size = {
+            let size = terminal::size()?;
+            // if reported size is 0, 0 - 
+            // use a default size to avoid divide by 0 panics
+            if size == (0, 0) {
+                (24, 80)
```

Comment:
Should be columns, rows
https://docs.rs/crossterm/latest/crossterm/terminal/fn.size.html
```suggestion
                (80, 24)
```